guides/installation.md

```diff
@@ -42,6 +43,8 @@ sudo swapon /swapfile
 * These are necessary to compile the CLI:
 * `sudo apt-get install build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev libyaml-dev`
 
+> You can also install amber using [Linuxbrew](http://linuxbrew.sh/) `brew install amber`
```

Comment:
Interesting, we should try [Nix](https://nixos.org/nix/) as well. I remember @manveru mentioned it on Mint channel ( [Mint is using it](https://www.mint-lang.com/install) )

Reply:
I guess we can create Debian packages instead using the new travis deploy? Not sure if we want to support many different package managers